airflow/providers/ssh/operators/ssh.py

```diff
@@ -43,7 +46,12 @@ class SSHOperator(BaseOperator):
     :type remote_host: str
     :param command: command to execute on remote host. (templated)
     :type command: str
-    :param timeout: timeout (in seconds) for executing the command. The default is 10 seconds.
+    :param conn_timeout: timeout (in seconds) for maintaining the connection. The default is 10 seconds.
```

Comment:
Maybe the same language here as is used on the `remote_host`? "If provided, it will replace..."